snmp/README.md

```diff
@@ -2,11 +2,37 @@
 
 ## Overview
```

Comment:
@sarina-dd  @ruthnaebeck Hi :wave:, we previously moved most of the content from integrations-core/snmp/README.md to  https://docs.datadoghq.com/network_monitoring/devices in this PR: https://github.com/DataDog/integrations-core/pull/7458

I thought we want to consolidate all snmp/ndm docs to https://docs.datadoghq.com/network_monitoring/devices . Is that correct ? If so, shall improve https://docs.datadoghq.com/network_monitoring/devices docs instead of updating integrations-core/snmp/README.md ?

(we still have some docs related to profile creation at https://datadoghq.dev/integrations-core/tutorials/snmp, but maybe it can make more sense for user to also have that in https://docs.datadoghq.com/network_monitoring/devices, WDYT?)

cc @naaltman

Reply:
It sounds like the final decision is we're keeping the SNMP overview on the integration page as that was the initial decision. Keeping this PR as-is and we can migrate in the future if needed.